src/user-projects-hooks.tsx

```diff
@@ -81,8 +80,8 @@ export const useMakeCodeProject = () => {
     return {
       [ProjectFilenames.MainTs]: generateMainScript(gestures, "javascript"),
       [ProjectFilenames.MainBlocks]: generateMainScript(gestures, "blocks"),
-      [ProjectFilenames.CustomTs]: generateCustomTs(gestures, model),
-      [ProjectFilenames.CustomJson]: generateCustomJson(gestures),
+      [ProjectFilenames.Autogenerated]: generateCustomTs(gestures, model),
+      [ProjectFilenames.DataJson]: generateCustomJson(gestures),
```

Comment:
This will be changed to `dataset.json` with all the gesture data in https://github.com/microbit-foundation/ml-trainer/pull/301.